### PR TITLE
Add the commercial activity document

### PR DESCRIPTION
**Pull request by royrapoport**
This adds the commercial activity document I wrote and vetted in #rls-culture.  Discussion of this document can be found [here](https://rands-leadership.slack.com/archives/C9Y28QLVC/p1701207989374139).

### CODE REVIEW

**Comment by arjitsrivastava** [comment]
Should we link this file in the main README doc as well -- https://github.com/randsleadershipslack/documents-and-resources/blob/main/README.md?

**Comment by royrapoport** [comment]
@arjitsrivastava I like that idea, once this is finalized and in the repo.  So I have a **very mild** preference to have that be a followup PR rather than this PR, but that's a personal pref -- do you think we should do that all as part of one PR?

File: commercial-activity.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Some Thoughts on "Commercial Activity" and Market Research
+**(December 2, 2023 / @royrapoport)**
+
+## Background
+
+As the CoC for RLS indicates, RLS has a pretty significant allergy to activity considered to be ‘commercial activity’.  However, as the CoC is relatively succinct, there’s not a lot of guidance as to how to think about "what commercial activity is," and how it will likely be received.  As a result, at some regular frequency we get into conversations in either #rls-rules or #rls-culture about whether or not some already-done or considered activity would be commercial.
+
+Some of these conversations (particularly this one and this one) have been useful for me to better understand the perspective of the community, and at some point I volunteered to try to write up a document that would capture our thoughts about commercial activity in a more conversational and explanatory way for other people that would allow them to understand how we think about this without having to trawl through many different threads.
```
Comment (globau) [inline commercial-activity.md]: > (particularly this one and this one)

As the "this one"s are unlinked this should probably be removed

**Comment by angelariggs** [comment]
This looks good! It got community approval, this is just codifying as part of the RLS docs system instead of a canvas in Slack to it's easier to link and access. Next step will be linking to the main CoC. Thanks @royrapoport!

**Pull request merged**